I did what is described here: https://github.com/ESMValGroup/ESMValTool_Tutorial/blob/main/CONTRIBUTING.md#previewing-your-changes-locally

```bash
cd ESMValTool_Tutorial
mamba env create -f environment.yml -n tut
mamba activate tut
make serve
```

With your changes, I get this now:

```bash
$ make serve
bundle install && bundle update && bundle exec jekyll serve
Bundle complete! 1 Gemfile dependency, 93 gems now installed.
Bundled gems are installed into `./.vendor/bundle`
[DEPRECATED] Pass --all to `bundle update` to update everything
Fetching gem metadata from https://rubygems.org/...........
Resolving dependencies...
Bundle updated!
/home/manuel/ESMValTool_Tutorial/.vendor/bundle/ruby/4.0.0/gems/jekyll-3.9.0/lib/jekyll.rb:28: warning: csv used to be loaded from the standard library, but is not part of the default gems since Ruby 3.4.0.
You can add csv to your Gemfile or gemspec to fix this error.
bundler: failed to load command: jekyll (/home/manuel/ESMValTool_Tutorial/.vendor/bundle/ruby/4.0.0/bin/jekyll)
/home/manuel/micromamba/envs/tut/lib/ruby/4.0.0/bundled_gems.rb:60:in 'Kernel.require': cannot load such file -- csv (LoadError)
        from /home/manuel/micromamba/envs/tut/lib/ruby/4.0.0/bundled_gems.rb:60:in 'block (2 levels) in Kernel#replace_require'
        from /home/manuel/ESMValTool_Tutorial/.vendor/bundle/ruby/4.0.0/gems/jekyll-3.9.0/lib/jekyll.rb:28:in '<top (required)>'
        from /home/manuel/micromamba/envs/tut/lib/ruby/4.0.0/bundled_gems.rb:60:in 'Kernel.require'
        from /home/manuel/micromamba/envs/tut/lib/ruby/4.0.0/bundled_gems.rb:60:in 'block (2 levels) in Kernel#replace_require'
        from /home/manuel/ESMValTool_Tutorial/.vendor/bundle/ruby/4.0.0/gems/jekyll-3.9.0/exe/jekyll:8:in '<top (required)>'
        from /home/manuel/micromamba/envs/tut/lib/ruby/4.0.0/rubygems.rb:304:in 'Kernel#load'
        from /home/manuel/micromamba/envs/tut/lib/ruby/4.0.0/rubygems.rb:304:in 'Gem.activate_and_load_bin_path'
        from /home/manuel/ESMValTool_Tutorial/.vendor/bundle/ruby/4.0.0/bin/jekyll:25:in '<top (required)>'
        from /home/manuel/micromamba/envs/tut/lib/ruby/4.0.0/bundler/cli/exec.rb:61:in 'Kernel.load'
        from /home/manuel/micromamba/envs/tut/lib/ruby/4.0.0/bundler/cli/exec.rb:61:in 'Bundler::CLI::Exec#kernel_load'
        from /home/manuel/micromamba/envs/tut/lib/ruby/4.0.0/bundler/cli/exec.rb:24:in 'Bundler::CLI::Exec#run'
        from /home/manuel/micromamba/envs/tut/lib/ruby/4.0.0/bundler/cli.rb:500:in 'Bundler::CLI#exec'
        from /home/manuel/micromamba/envs/tut/lib/ruby/4.0.0/bundler/vendor/thor/lib/thor/command.rb:28:in 'Bundler::Thor::Command#run'
        from /home/manuel/micromamba/envs/tut/lib/ruby/4.0.0/bundler/vendor/thor/lib/thor/invocation.rb:127:in 'Bundler::Thor::Invocation#invoke_command'
        from /home/manuel/micromamba/envs/tut/lib/ruby/4.0.0/bundler/vendor/thor/lib/thor.rb:538:in 'Bundler::Thor.dispatch'
        from /home/manuel/micromamba/envs/tut/lib/ruby/4.0.0/bundler/cli.rb:35:in 'Bundler::CLI.dispatch'
        from /home/manuel/micromamba/envs/tut/lib/ruby/4.0.0/bundler/vendor/thor/lib/thor/base.rb:584:in 'Bundler::Thor::Base::ClassMethods#start'
        from /home/manuel/micromamba/envs/tut/lib/ruby/4.0.0/bundler/cli.rb:29:in 'Bundler::CLI.start'
        from /home/manuel/micromamba/envs/tut/lib/ruby/gems/4.0.0/gems/bundler-4.0.6/exe/bundle:28:in 'block in <top (required)>'
        from /home/manuel/micromamba/envs/tut/lib/ruby/4.0.0/bundler/friendly_errors.rb:118:in 'Bundler.with_friendly_errors'
        from /home/manuel/micromamba/envs/tut/lib/ruby/gems/4.0.0/gems/bundler-4.0.6/exe/bundle:20:in '<top (required)>'
        from /home/manuel/micromamba/envs/tut/lib/ruby/4.0.0/rubygems.rb:304:in 'Kernel#load'
        from /home/manuel/micromamba/envs/tut/lib/ruby/4.0.0/rubygems.rb:304:in 'Gem.activate_and_load_bin_path'
        from /home/manuel/micromamba/envs/tut/bin/bundle:31:in '<main>'
make: *** [Makefile:49: serve] Error 1
```

--- a/Gemfile
+++ b/Gemfile
@@ -8,3 +8,6 @@ git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
 ruby '>=2.5.3'
 
 gem 'github-pages', group: :jekyll_plugins
+gem 'jekyll', '~> 4.3'
+gem 'csv'
+gem 'bigdecimal'

--- a/Gemfile
+++ b/Gemfile
@@ -7,7 +7,9 @@ git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
 # Synchronize with https://pages.github.com/versions
 ruby '>=2.5.3'
 
-gem 'github-pages', group: :jekyll_plugins
+#  github-pages conflics with jekyll+liquid, but the site
+#  builds well locally
+#  gem 'github-pages', group: :jekyll_plugins
 gem 'jekyll', '~> 4.3'
 gem 'csv'
 gem 'bigdecimal'

--- a/Makefile
+++ b/Makefile
@@ -7,7 +7,7 @@ export SHELL = /bin/bash
 # Settings
 MAKEFILES=Makefile $(wildcard *.mk)
 JEKYLL_VERSION=3.8.5
-JEKYLL=bundle install && bundle update && bundle exec jekyll
+JEKYLL=bundle install && bundle update --all && bundle exec jekyll
 PARSER=bin/markdown_ast.rb
 DST=_site
 

